screens/Home.js

```diff
@@ -93,67 +107,55 @@ class Home extends React.Component {
       }
     } else if (prevProps.statusBar !== statusBar) {
       this.changeStatusBar(statusBar);
-    } else if (prevProps.statistics !==statistics) {
+    } else if (prevProps.statistics !== statistics) {
       AnalyticsManager.getInstance().allowTracking(statistics);
     } else if (prevProps.reminders !== reminders) {
       NotificationsManager.getInstance().checkPermissions(reminders);
-      NotificationsManager.getInstance().updateReminders(
-        reminders,
-        reminderSound,
-        reminderBanis
-      );
+      NotificationsManager.getInstance().updateReminders(reminders, reminderSound, reminderBanis);
     } else if (prevProps.reminderSound !== reminderSound) {
-      NotificationsManager.getInstance().updateReminders(
-        reminders,
-        reminderSound,
-        reminderBanis
-      );
+      NotificationsManager.getInstance().updateReminders(reminders, reminderSound, reminderBanis);
 
       if (actions.REMINDER_SOUNDS.indexOf(reminderSound) !== 0) {
-        const sound = new Sound(
-          reminderSound,
-          Sound.MAIN_BUNDLE,
-          (error) => {
-            if (error) {
-              return;
-            }
-            // loaded successfully
-            sound.play(() => {
-              sound.reset();
-            });
+        const sound = new Sound(reminderSound, Sound.MAIN_BUNDLE, (error) => {
+          if (error) {
+            return;
           }
-        );
+          // loaded successfully
```

Comment:
I think this code is self-explanatory veerji. You can remove this comment.

Reply:
fixed